org.opent2t.sample.lamp.superpopular/com.smartthings.lightbulb/js/thingTranslator.js

```diff
@@ -194,8 +194,8 @@ function providerSchemaToPlatformSchema(providerSchema, expand) {
             controlId: controlId
         },
         pi: providerSchema['id'],
-        mnmn: providerSchema['manufacturer'],
-        mnmo: providerSchema['model'],
+        mnmn: providerSchema['manufacturer'] === null ? "Unkown" : providerSchema['manufacturer'],
+        mnmo: providerSchema['model'] === null ? "Unkown" : providerSchema['model'],
```

Comment:
Changed to return 'Undefined'.

---
In reply to: [92027584](https://github.com/openT2T/translators/pull/134#discussion_r92027584) [](ancestors = 92027584)